Server.js

```diff
@@ -0,0 +1,24 @@
+const express=require('express');
+const app=express();
+const bodyParser = require('body-parser')
+const dbconnection = require('./dbconnection');
+const userRouter=require('./userRouter');
+
+dbconnection();
+
+app.use(bodyParser.json());
+
+
+app.use('/api',userRouter);
+
+/*
+app.use('/',(req,res)=>
+{
+    res.send("i am saranya")
+})
+*/
+//to create the port
+app.listen(3000,()=>
+{
+  console.log("server start")
```

Comment:
formatting not proper and message should not be generic

`Folks app server started at port ${port} at ${new Date()}`